tests/test_browser.go

```diff
@@ -82,6 +84,16 @@ func newTestBrowser(tb testing.TB, opts ...any) *testBrowser {
 		vu.CtxField = ctx
 	}
 
+	// This is used to help the browserprocess register
+	// distinguish between different error runs so that
+	// a panic in one test doesn't force close the browser
+	// in other running tests.
+	u, err := uuid.NewV4()
```

Comment:
I could use a global atomic counter instead of `uuid` so that we're not having to depend on another library, WDYT?

Reply:
I think the `uuid` library will be required anyway because it's included in k6, but I think an atomic counter will make it simpler so +1 for me.